Joinmarket-messaging-protocol.md

```diff
@@ -200,4 +201,66 @@ Commitment data for default PoDLE construct:
 
 **(NS)**: nick signature (either of form pub, sig or from pubkey recovery, bitcoin type) : message to be signed is the whole message to be sent + message channel identifier str(`serverport`) (the latter to prevent cross-channel replay).
 
+
+Fidelity Bonds
+==========
+
+JoinMarket version 0.9 implements fidelity bonds, for a full writeup of whys and hows of the feature see [the design document](https://gist.github.com/chris-belcher/18ea0e6acdb885a2bfbdee43dcd6b5af/).
+
+Makers who have a fidelity bond will send a proof of it to takers in response to an !orderbook message along with the maker's offers.
+
+The fidelity bond proof message contains everything needed to prove to the taker that the fidelity bond UTXO is real, that it has a certain value, and that it is genuinely possessed by the maker.
+To keep the size down the fidelity bond proof message is a base64-encoded binary data blob.
+
+The binary blob is made up of concatenating various parameters (size in bytes on the line below):
+
+```
+nick_sig + cert_sig + cert_pubkey + cert_expiry + utxo_pubkey + txid + vout + timelock
+72       + 72       + 33          + 2           + 33          + 32   + 4    + 4 = 252 bytes
+```
+
+The command name "tbond" means "time locked bond", one day there might be a burned coins fidelity bond too, but probably not because timelocked bonds 
+
+### UTXO data
+To prove that the UTXO exists, the proof contains the `txid` and `vout` of the UTXO, these can be used to lookup in a full node's UTXO set to check that the UTXO exists, is confirmed in the strongest chain and is unspent.
```

Comment:
not important but don't we usually say 'heaviest' chain or most-proof-of-work chain?

Reply:
I'm going to switch to "best" because Core uses it in the name `getbestblockhash`